src/validation.cpp

```diff
@@ -1017,7 +1017,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
             .descendant_count = maybe_rbf_limits.descendant_count + 1,
             .descendant_size_vbytes = maybe_rbf_limits.descendant_size_vbytes + EXTRA_DESCENDANT_TX_SIZE_LIMIT,
         };
-        if (ws.m_vsize > EXTRA_DESCENDANT_TX_SIZE_LIMIT || ws.m_ptx->nVersion == 3) {
```

Comment:
few remaining spots that I can see:
```
src/test/fuzz/package_eval.cpp:176:                tx_mut.nVersion = fuzzed_data_provider.ConsumeBool() ? 3 : CTransaction::CURRENT_VERSION;
src/test/fuzz/tx_pool.cpp:229:            tx_mut.nVersion = fuzzed_data_provider.ConsumeBool() ? 3 : CTransaction::CURRENT_VERSION;
src/test/txvalidation_tests.cpp:289:        mtx_many_sigops.nVersion = 3;
src/test/util/txmempool.cpp:121:        if (tx_info.tx->nVersion == 3) {
src/test/util/txmempool.cpp:136:                Assert(parents.begin()->get().GetSharedTx()->nVersion == 3);
src/test/util/txmempool.cpp:141:                Assert(parent.get().GetSharedTx()->nVersion != 3);
```

can also use this constant

Reply:
thanks! fixed these now